src/table/paginator.scss

```diff
@@ -70,4 +71,9 @@
   > .iui-right {
     justify-content: flex-end;
   }
+
+  &-page-size-label {
+    @include iui-text-muted;
+    white-space: nowrap;
```

Comment:
Doesn't this also need `overflow: hidden` and `text-overflow: ellipsis`?

Reply:
I thought we were planning to use React to auto-hide the label when the width gets too narrow?  If that's the case, truncation should never happen.

Reply:
Correct, we're hiding the label at a certain width (currently 1024px but maybe needs to be more narrow). But it can still overflow. See https://github.com/iTwin/iTwinUI-react/pull/427#discussion_r752500421

Reply:
Ok, makes sense.  I've added this.